spring-file/jmx-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Run performance tests for Spring Boot server locally
+
```

Comment:
Since you use relative paths here, I'd add `cd ${0%/*}` as first command to `cd` into script directory